client/src/Movies/MovieList.js

```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
```

Comment:
Good, so `'react'` has a default export of some big object. We're importing that object and calling it `React`. So if we want to use one of the key/value pairs on that object, like `Component`, we do `React.Component`, right? So for example:

`class SomeClass extends React.Component` because we want all of our components to be React Components. So if we want, we can destructure that as it comes in just like you did with your code and say:
`import { Component } from "react"`

See?! You're already doing this, I just want to reiterate it to demystify it. You're doing the same thing on line three.